deploy/VerifierRayTracing10Private.ts

```diff
@@ -0,0 +1,17 @@
+import { DeployFunction } from "hardhat-deploy/types";
+
+const func: DeployFunction = async ({
+  deployments: { deploy },
+  getNamedAccounts,
+}) => {
+  const { deployer } = await getNamedAccounts();
+  await deploy("VerifierRayTracing10Private", {
```

Comment:
Looks like this string is the only difference between this file and VerifierRayTracing4Private.ts. Could it be simplified?

Reply:
Yes, I think this and the .circom files used as the base for the circuits could be merged into one and make this a variable. 
Maybe next PR?